core/src/types/mod.rs

```diff
@@ -24,7 +24,7 @@ mod tests {
 		inputs::*,
 		models::{
 			epub::*, library::*, list_directory::*, log::*, media::*, read_progress::*,
-			series::*, tag::*, user::*,
+			series::*, tag::*, user::*, readinglist::*
```

Comment:
Can you add the type generation for the two new models, as well? You can place it underneath the one for `ReadProgress` or something. Would essentially just be:

```rust
file.write_all(format!("{}\n\n", ts_export::<ReadingList>()?).as_bytes())?;
file.write_all(format!("{}\n\n", ts_export::<CreateReadingList>()?).as_bytes())?;
```